portfolio/src/main/webapp/visualizer.js

```diff
@@ -4,23 +4,41 @@ google.charts.load('current', {
 });
 google.charts.setOnLoadCallback(drawRegionsMap);
 
-function drawRegionsMap() {
-  const data = google.visualization.arrayToDataTable([
-    ['City',   'Population', 'Area'],
-    ['Chicago', 2000000, 1285.31],
-    ['New York City', 5000000, 1400.31],
-    ['Los Angeles', 4000000, 1300],
-    ['Philadelphia', 1500000, 800],
-    ['Stamford', 30000, 900],
-  ]);
 
+function drawRegionsMap(i) {
+  if (typeof(i) == "undefined") {
+    i = 0;
+  }
   const options = {
     region: 'US',
-    displayMode: 'markers',
     colorAxis: {colors: ['white', 'green']},
+    displayMode: 'regions',
+    resolution: 'provinces', 
     width: 900,
-    height: 500
+    height: 500,
+    animation:{
+      duration: 1000,
+      easing: 'out',
+    }
   };
-  const chart = new google.visualization.GeoChart(document.getElementById('regions_div'));
-  chart.draw(data, options);
+
+  let data = new google.visualization.DataTable();
+  data.addColumn('string', 'State');
+  data.addColumn('number', 'Cases');
+
+  fetch('/covid-data').then(response => response.json()).then((covidData) => {
+    let curr = covidData[i].cases;
```

Comment:
`let` --> `const`

Reply:
Maybe rename `curr` --> `dailyCovidCasesByState`? Not entirely clear what curr represents.